covasim/webapp/index.html

```diff
@@ -14,7 +14,7 @@
   <script src="https://polyfill.io/v3/polyfill.min.js?version=3.52.1&features=es2015%2CIntersectionObserver"></script>
   <script src="https://unpkg.com/vue@2.6.11/dist/vue.min.js"></script>
   <script src="https://unpkg.com/bootstrap-vue@2.11.0/dist/bootstrap-vue.min.js"></script>
-  <link href="https://fonts.googleapis.com/css?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
```

Comment:
Quicksand is hard to read on non-Ultra HD monitors. Please pick another font or choose a heavier default weight.

Reply:
I changed to 
```
<link href="https://fonts.googleapis.com/css?family=Quicksand:wght@600;900&display=swap" rel="stylesheet">
```
(is that right?) Can you share a screenshot of the low-res readability issue? It looked fine on my phone

Reply:
I can't share a screenshot as your monitor is the issue. I would have to take a photo of the monitor itself. 
I was looking at it on a 1080p monitor on windows (the most common resolution on the internet). Thin fonts just don't render as well and most phones nowadays have higher res screens than that.

Reply:
This is maybe overkill but the useragent has screen resolution info doesn't it? Could pick the font responsively

Reply:
Or if Windows is detected, set the font to Courier New with 8-bit colors ;)

Reply:
That is more work than is necessary. The updated URL is correct, but you'll need to override the body font-weight to 600 too.

Reply:
Picking a font based on useragent is a nightmare waiting to happen because useragents lie all the time.

Reply:
that's true, if you're lying about your screen resolution then maybe you deserve what's coming ;) tried heavier weight, it's...not great: 
![image](https://user-images.githubusercontent.com/53099813/79920761-afe52180-83e5-11ea-9cdb-f2e00c1b2a8b.png)
suggest leaving for now. we do have a user group who we could ask, and can certainly revisit if it comes up

Reply:
Added to issue #180